packages/styletron-react/src/__tests__/tests.browser.js

```diff
@@ -368,6 +368,64 @@ test("ref forwarding", t => {
   Enzyme.mount(<TestComponent />);
 });
 
+test("React.createRef() ref forwarding", t => {
+  t.plan(1);
+
+  const Widget = styled("button", {color: "red"});
+  class TestComponent extends React.Component<{}> {
+    widgetInner: {current: null | HTMLButtonElement};
```

Comment:
Flow seems unhappy with this.

Reply:
Did a fix but not great. Not entirely sure how to type the forwardRef thing (which seems to be the real problem).